addons/binding/org.openhab.binding.tplinksmarthome/README.md

```diff
@@ -0,0 +1,133 @@
+
+* Switch On/Off
+* Adjust light appearance from soft white (2700k) to daylight (6500k)
+* Adjust the brightness
+* Wi-Fi signal strength (rssi)
+Switching and Brightness is done using the `brightness` channel.
+
+### LB130 Smart Wi-Fi LED Bulb with Color Changing Hue
+
+* Switch On/Off
+* Fine-tune colors
+* Adjust light appearance from soft white (2500k) to daylight (9000k)
+* Adjust the brightness
+* Wi-Fi signal strength (rssi)
+Switching, Brightness and Color is done using the `color` channel.
+
+### HS105 and HS100 Smart Wi-Fi Plug
+
+* Switch On/Off
+* Wi-Fi signal strength (rssi)
+
+### HS110 Smart Wi-Fi Plug
+
+* Switch On/Off
+* Energy readings
+* Wi-Fi signal strength (rssi)
+
+### HS200 Smart Wi-Fi Switch
+
+* Switch On/Off
+The default refresh is set to 1 second. So it polls the switch for status changes. If you don't use the switch manually
+often, you can set it to a higher refresh. The refresh is only relevant to detect manual using the switch. Switching
+via openHAB activates the switch directly.
+* Wi-Fi signal strength (rssi)
+
+### RE270K AC750 Wi-Fi Range Extender with Smart Plug
+
+* Switch On/Off
+* Wi-Fi signal strength (rssi)
+
+### RE370K AC1200 Wi-Fi Range Extender with Smart Plug
+
+* Switch On/Off
+* Wi-Fi signal strength (rssi)
+
+## Prerequisites
+
+Before using Smart Plugs with openHAB the devices must be connected to the Wi-Fi network. This can be done using the
+TP-Link provided mobile app Kasa.
+
+## Discovery
+
+Devices can be auto discovered in the same local network as the openHAB application. It's possible to connect to
+devices in a different network, but these must be added manually.
+
+## Thing Configuration
+
+The thing id is the product type in lower case. For example `HS100` has id `hs100`.
+
+The thing has a few configuration parameters:
+
+| Parameter          | Description                                                                 |
+|--------------------|-----------------------------------------------------------------------------|
+| ipAddress          | IP Address of the device. Mandatory.                                        |
+| refresh            | Refresh interval in seconds. Optional, the default value is 30 seconds.     |
+| transitionPeriod   | Duration of state changes in milliseconds, only for light bulbs, default 0. |
+
+## Channels
+
+All devices support some of the following channels:
+
+| Channel Type ID  | Item Type | Description                                    | Thing types supporting this channel                             |
+|------------------|-----------|------------------------------------------------|-----------------------------------------------------------------|
+| switch           | Switch    | Switch the Smart Home device on or off.        | HS100, HS105, HS110, RE270K, RE370K                             |
+| brightness       | Switch    | Switch the Smart Home device on or off.        | LB100, LB110, LB120                                             |
```

Comment:
remove this line because the channel type IS marked as a channel for "Dimmer" items (which always then has to work on Switch items as well).